### PR TITLE
Gathering information about donors on the thank you page

### PR DESCRIPTION
**Pull request by olasitarska**
I am not sure about uploading logos -- should we make some restrictions here? where to save them?

### CODE REVIEW

**Comment by asendecka** [comment]
It looks good to me - I think we should merge it.

File: djangoproject/templates/fundraising/thank-you.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  fill out the form below. This is, of course, optional.</p>
+
+<h1>You're now a Django Hero:</h1>
+<form action="" method="post" class="form-input">
+  {% csrf_token %}
+  {% for field in form %}
+    {% if field.errors %}<p class="errors">{{ field.errors.as_text }}</p>{% endif %}
+    <p>
+      {{ field }}
+      {% if field.id_for_label == 'id_is_visible' or field.id_for_label == 'id_is_subscribed' or field.id_for_label == 'id_is_amount_displayed' %}
```
Comment (timgraham) [inline djangoproject/templates/fundraising/thank-you.html]: Could we add an attribute with a explanatory name to the form with these field names and check `if field.name in form.FIELDS instead?

File: fundraising/forms.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+from .models import DjangoHero
+
+class AddDjangoHeroForm(forms.ModelForm):
+    email = forms.CharField(required=True, widget=forms.TextInput(attrs={'class': 'required', 'placeholder': 'E-mail address'}),
+        help_text='We ask for email because we need to match your donation with information from this form.')
+    name = forms.CharField(required=False, widget=forms.TextInput(attrs={'class': 'required', 'placeholder': 'Your name or name of your organization'}))
+    url = forms.CharField(required=False, widget=forms.TextInput(attrs={'placeholder': 'Should we link your name to somewhere? Tell us where'}))
+    logo = forms.FileField(required=False, help_text='If you donated at least $200, you can submit your logo and we will display it too.')
+    is_visible = forms.BooleanField(required=False, label='Yes, I want this informations to be displayed on the Fundraising page.')
+    is_subscribed = forms.BooleanField(required=False, label='Django Software Foundation can ocasionally inform me about fundraising.')
```
Comment (timgraham) [inline fundraising/forms.py]: occasionally

File: fundraising/views.py [hunk 3]
```diff
@@ -1,3 +1,18 @@
...
+from django.shortcuts import render, redirect
+from django.core.urlresolvers import reverse
+from django.contrib import messages
 
-# Create your views here.
+from .forms import AddDjangoHeroForm
+
+def fundraising_index(request):
+    return render(request, 'fundraising/index.html', {})
+
```
Comment (timgraham) [inline fundraising/views.py]: two newlines for flake8 (looks like a cleanup is needed on master)

File: fundraising/views.py [hunk 4]
```diff
@@ -1,3 +1,19 @@
-from django.shortcuts import render
+from django.shortcuts import render, redirect
```
Comment (timgraham) [inline fundraising/views.py]: please alphabetize imports

File: fundraising/forms.py [hunk 5]
```diff
@@ -0,0 +1,18 @@
+from django import forms
+
+from .models import DjangoHero
+
+
+class AddDjangoHeroForm(forms.ModelForm):
+    email = forms.CharField(required=True, widget=forms.TextInput(attrs={'class': 'required', 'placeholder': 'E-mail address'}),
```
Comment (timgraham) [inline fundraising/forms.py]: suggested style so lines are not so long:
```
forms.CharField(
    required=True,
    widget=...,
)

File: djangoproject/templates/fundraising/thank-you.html (hunk 1 above)
Comment (asendecka) [inline djangoproject/templates/fundraising/thank-you.html]: I am not sure if I understand it: you mean defining FIELDS on form as:
`FIELDS = ['is_visible', 'is_subscribed', 'is_amount_displayed']` and then do a check in template as you suggested? Or am I missing something? :)

Comment (timgraham) [inline djangoproject/templates/fundraising/thank-you.html]: Correct, with a more descriptive name, maybe something like `THANK_YOU_PAGE_FIELDS`?

Comment (timgraham) [inline djangoproject/templates/fundraising/thank-you.html]: (I'm not quite sure the purpose of this check, so the suggested name might be wrong.)

Comment (asendecka) [inline djangoproject/templates/fundraising/thank-you.html]: They are checkboxes - we want to display labels for them (contrary to the standard inputs, where we display it inside an input). So I think `CHECKBOX_FIELDS` should be good enough (or `THANK_YOU_CHECKBOX_FIELDS`)

**Comment by olasitarska** [comment]
Thanks @asendecka for fixing my PR! :) :heart_eyes_cat:

**Comment by asendecka** [comment]
@olasitarska I want it merged :), so I can work on other parts without worrying too much :) :computer:

**Comment by asendecka** [comment]
I'm merging it :)!

**Pull request merged**